src/XrdCl/XrdClZipArchiveReader.cc

```diff
@@ -243,7 +243,7 @@ class ZipArchiveReaderImpl
     {
       delete pBuffer;
       ClearRecords();
-      pArchive.Close();
+      XRootDStatus st = pArchive.Close();
```

Comment:
The declaration of `Close()` has an attribute to check return value, but I could't figure out what to do (we are in destructor). Just saving return value tricks the compiler to avoid the warning, but probably is not the right approach.